toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss

```diff
@@ -0,0 +1,39 @@
+.js {
+    .c-submission-details-header {
+        box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.2);
+
+        &__row,
+        &__author-list {
+            display: flex;
+        }
+
+        &__row-item {
+            &:not(:last-child) {
+                &:after {
+                    content: ' |';
+                    padding-left: 10px;
+                }
+            }
+        }
+    }
+
+    .abstract-expander {
+        @include arrow(map-get($context--colors, deep-green), down, 4px);
+
+        padding: 0;
+        margin: 0;
+        text-align: left;
+        min-width: 105px;
```

Comment:
this is probably a personal preference thing but i quite like having exact value / config like this stored as a scss variable in 10-settings so that devs in the future can easily configure a component from the 10-settings file

Reply:
Yeah, I had similar thoughts. I usually don't bother unless the CSS properties are linked, reused, or likely to change, or dependent on context.